ChangeLog

```diff
@@ -1,3 +1,10 @@
+2024-10-29 Xanthos Xanthopoulos <xanxanthopoulos@gmail.com>
```

Comment:
I uses two whitespaces here between date and name, and name and email, but I can touch that up.  Same for line length.